src/main.c

```diff
@@ -1531,18 +1553,19 @@ static int on_config_listen(h2o_configurator_command_t *cmd, h2o_configurator_co
     yoml_t **ssl_node = NULL, **owner_node = NULL, **permission_node = NULL, **quic_node = NULL, **cc_node = NULL,
            **initcwnd_node = NULL;
     int proxy_protocol = 0;
+    int packet_timestamping_enabled = 0;
```

Comment:
```suggestion
    int proxy_protocol = 0, packet_timestamping = 0;
```
`proxy_protocol` is a boolean, that turns on / off the feature. We can follow that convention by calling the new variable `packet_timestamping`.

Reply:
sounds good, thanks.